src/components/_global/BalModal/BalModal.vue

```diff
@@ -1,5 +1,5 @@
 <script lang="ts" setup>
```

Comment:
Changes to this file are to prevent background scrolling while modals are present